client/src/types/boost.ts

```diff
@@ -1,10 +1,114 @@
-export type BoostConfigType = 'powerPlan' | 'rider' | 'bike' | 'track' | 'all';
+import {
+  Number,
+  String,
+  Array,
+  Record,
+  Union,
+  Runtype,
+  Static,
+} from 'runtypes';
+
+export type BoostConfigType =
+  | 'powerPlan'
+  | 'rider'
+  | 'bike'
+  | 'track'
+  | 'bundle';
+
+type configName = {
+  displayName: string;
+  fileName: string;
+};
 
 export interface BoostConfig {
   /** The input of BOOST that this config is for */
   type: BoostConfigType;
   /** List of available BOOST configuration files */
-  options: string[];
+  options: configName[];
   /** Currently selected BOOST configuration */
-  active?: string;
+  active?: configName;
```

Comment:
We use the `displayName` attribute to display the currently selected config in the Accordion header. (See line 129 in `BoostConfigurator`). I'm also a bit unsure of what you mean by the edge case where the display name doesn't match.

Reply:
Can always change this the accordion to work off the filename instead! I reckon that's better, since the file will be unique, but the display name may not be.

The edge case I was referring to was where a display name and file name were provided that don't actually match any pair of configs.

Reply:
That's good point, but imo having filename in the header won't be as aesthetic or user friendly (I assume users would remember display names better than file names and hence being able to see `Wombat` rather than `v2.json` would be more ideal). This is an assumption I'm making, but you would probably have a better idea to whether this assumption would hold with people in MHP. Let me know!